### PR TITLE
popcorn: enhance ConsoleLog

### PR DESCRIPTION
**Pull request by Morriar**
Add some logger capabilities to ConsoleLog so it can be shared with the rest of the popcorn app as a classic logger.

### CODE REVIEW

File: lib/popcorn/pop_middlewares.nit [hunk 1]
```diff
@@ -42,23 +42,65 @@ end
 class ConsoleLog
 	super Handler
 
+	# Logger level
+	#
+	# * `1`: errors
+	# * `2`: warnings
+	# * `3`: info
+	# * `4`: debug
```
Comment (privat) [inline lib/popcorn/pop_middlewares.nit]: `0` should be documented.

**Comment by privat** [comment]
+1

File: lib/popcorn/pop_middlewares.nit [hunk 2]
```diff
@@ -42,23 +42,65 @@ end
...
+	#
+	# Request status are always logged, whatever the logger level is.
+	var level = 4 is writable
```
Comment (jcbrinfo) [inline lib/popcorn/pop_middlewares.nit]: 0 is missing.

PS: I hope we will have enumerated type soon… :sweat_smile:

File: lib/popcorn/pop_middlewares.nit (hunk 1 above)
Comment (jcbrinfo) [inline lib/popcorn/pop_middlewares.nit]: I thought of the same thing.

PS: I hope we will have enumerated type soon… :sweat_smile:

**Comment by Morriar** [comment]
reroll

File: lib/popcorn/pop_middlewares.nit [hunk 3]
```diff
@@ -42,23 +42,68 @@ end
 class ConsoleLog
 	super Handler
 
+	# Logger level
+	#
+	# * `0`: silent
```
Comment (jcbrinfo) [inline lib/popcorn/pop_middlewares.nit]: It seems to be the complete opposite (the loudest level), isn’t it?

Comment (Morriar) [inline lib/popcorn/pop_middlewares.nit]: What do you mean?

Comment (jcbrinfo) [inline lib/popcorn/pop_middlewares.nit]: Oups! I was confusing the “logger”’s level with the message’s level. Sorry. My mistake.

**Comment by jcbrinfo** [comment]
:shipit:

**Pull request merged**